### PR TITLE
change the replicas keyword to service_num

### PR DESCRIPTION
**Pull request by tiansuo114**
change the replicas keyword in topoloy.yaml to service_num
but also keep the old version,old vversion topoloy.yaml still can use

other:
Change the replicas keyword displayed after executing the curveadm status command to service num

将 topoloy.yaml 中的replicas关键字更改为service_num
但也保留旧版本，旧版本的topoloy.yaml仍然可以使用

其他：
将执行curveadm status命令后显示的replicas关键字改为service num

### CODE REVIEW

**Comment by Wine93** [comment]
I think maybe `instance` is more better, it's more meaningful and easy to understand.

**Comment by tiansuo114** [comment]
> I think maybe `instance` is more better, it's more meaningful and easy to understand.

Okay, I have completed the code change (that is, changed the service_num keyword to instance) and committed the code. Please help conduct a code review for my code.

File: cli/command/migrate.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func checkMigrateTopology(curveadm *cli.CurveAdm, data string) error {
 		dcs2add[0].GetRole() != dcs2del[0].GetRole() {
 		return errno.ERR_REQUIRE_SAME_ROLE_SERVICES_FOR_MIGRATING
 	}
-	if len(dcs2del) != dcs2del[0].GetReplicas() {
+	if len(dcs2del) != dcs2del[0].GetInstance() {
```
Comment (Wine93) [inline cli/command/migrate.go:155]: Sorry, i didn't express clearly, actually we should use word instance**s** for multi-instances.

File: cli/command/status.go [hunk 2]
```diff
@@ -47,12 +47,13 @@ var (
...
 )
 
+// todo change status print from replicas to service num
 type statusOptions struct {
 	id           string
 	role         string
 	host         string
 	verbose      bool
-	showReplicas bool
+	showInstance bool
```
Comment (Wine93) [inline cli/command/status.go:56]: ditto.

File: internal/errno/errno.go [hunk 3]
```diff
@@ -348,11 +348,13 @@ var (
 	ERR_RENDERING_VARIABLE_FAILED       = EC(330007, "rendering variable failed")
 	ERR_CREATE_HASH_FOR_TOPOLOGY_FAILED = EC(330008, "create hash for topology failed")
 	// 331: configure (topology.yaml: invalid configure value)
-	ERR_UNSUPPORT_CLUSTER_KIND             = EC(331000, "unsupport cluster kind")
-	ERR_NO_SERVICES_IN_TOPOLOGY            = EC(331001, "no services in topology")
-	ERR_REPLICAS_REQUIRES_POSITIVE_INTEGER = EC(331002, "replicas requires a positive integer")
-	ERR_INVALID_VARIABLE_SECTION           = EC(331003, "invalid variable section")
-	ERR_DUPLICATE_SERVICE_ID               = EC(331004, "service id is duplicate")
+	//todo: remove ERR_REPLICAS_REQUIRES_POSITIVE_INTEGER and change to ERR_NO_SERVICES_NUM_TOPOLOGY
```
Comment (Wine93) [inline internal/errno/errno.go:351]: We should update the error code which show below.

**Comment by Wine93** [comment]
BTW: please rebase all commit into one when all is ready.

**Comment by tiansuo114** [comment]
I think i make a mistake in pull request😢😢
So i decide to make a new pull request,sorry to border

**Pull request closed without merging**